src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+import { Todo, User } from '../../types';
+import usersFromServer from '../../api/users';
```

Comment:
Avoid directly importing `usersFromServer` here. Instead, use the `users` prop passed to the component to ensure flexibility and consistency with the component's props.